src/transformers/models/detr/configuration_detr.py

```diff
@@ -155,6 +155,10 @@ def __init__(
         bbox_loss_coefficient=5,
         giou_loss_coefficient=2,
         eos_coefficient=0.1,
+        in_chans=3,
+        pretrained=True,
+        freeze_layers=True,
```

Comment:
I would leave away this one, as there's already a method one can call on `DetrModel`, called `freeze_backbone` as seen [here](https://github.com/huggingface/transformers/blob/705ca7f21b2b557e0cfd5d0853b297fa53489d20/src/transformers/models/detr/modeling_detr.py#L1183).

Maybe we can improve its documentation for visibility.

Reply:
I think this parameter is more specific, it disables freezing 2-4 layer of resnet-50 (this was just hard coded into encoder initialisation), but I can change it if you think freeze_backbone is better anyway